samples/snippets/src/test/java/dialogflow/cx/ExampleIT.java

```diff
@@ -45,13 +45,15 @@
   public void beforeTest() throws IOException {
     MockitoAnnotations.initMocks(this);
 
-    // use an empty string as the default request content
-    BufferedReader reader = new BufferedReader(new StringReader(""));
-    when(request.getReader()).thenReturn(reader);
+    stdOut = new ByteArrayOutputStream();
```

Comment:
nit: a safer way is to save stdout and then set to previous stdout https://github.com/GoogleCloudPlatform/java-docs-samples/blob/1cdb5921db12a0e30054932b9ca34be27900a1bb/storage-transfer/src/test/java/com/google/cloud/storage/storagetransfer/samples/test/ITStoragetransferSamplesTest.java#L307

Reply:
i removed the stdout to use something else, since this is a cloud function i'm using this [sample](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/functions/helloworld/hello-http-gradle/src/test/java/functions/HelloHttpTest.java) as an example.